naga/src/front/wgsl/lower/mod.rs

```diff
@@ -2347,6 +2347,42 @@ impl<'source, 'temp> Lowerer<'source, 'temp> {
                             );
                             return Ok(Some(result));
                         }
+                        "imageAtomicMin" | "imageAtomicMax" => {
+                            let mut args = ctx.prepare_args(arguments, 3, span);
+
+                            let image = args.next()?;
+                            let image = self.expression(image, ctx)?;
+
+                            let coordinate = self.expression(args.next()?, ctx)?;
+
+                            let ty = ctx
+                                .ensure_type_exists(crate::TypeInner::Scalar(crate::Scalar::I32));
+                            // We fib in a zero value for sample because it is not supported
+                            let sample =
+                                ctx.append_expression(crate::Expression::ZeroValue(ty), span)?;
```

Comment:
the reason i include it at all is because I dont know how to create a ZeroValue expression later on at the write stage